controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -15,6 +17,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	// secretKey is the key to the csi provisioner secret name
+	secretKey = "rook-csi-%s-provisioner"
```

Comment:
The name is very vague, pls change the name.

Reply:
csiProvisionerSecretName?